app/gui/view/welcome-screen/src/lib.rs

```diff
@@ -122,7 +120,7 @@ impl Model {
         let root = web::create_div();
         root.set_class_name(css_class::TEMPLATES_VIEW_ROOT);
         // We explicitly enable pointer events for Welcome Screen elements. Pointer events are
-        // disabled for all DOM layers by default.
+        // disabled for all DOM layers by default. See `DomLayers` documentation.
```

Comment:
```suggestion
        // disabled for all DOM layers by default. See [`DomLayers`] documentation.
```
This way `DomLayers` become a link on generated rustdoc (if the name is visible in the scope). See https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html

Reply:
Done